src/HealthCheck/HealthCheck/src/ClickView.Extensions.HealthCheck.csproj

```diff
@@ -1,16 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-    <Version>1.4.0-rc1</Version>
-    <LangVersion>7.1</LangVersion>
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```

Comment:
Instead of `net462`, should we make it `net471` since the minimum full fat we are using is .net 4.71 (domain big boy)?

Reply:
I just set it to the minimum that the rest of the dotnet runtime is set to ([see here](https://github.com/dotnet/runtime/blob/d82cac48bfa5d64fc94fc4b881d9214c9064c7d5/Directory.Build.props#L106)). Also 4.6.2 looks to still be supported (see [this](https://devblogs.microsoft.com/dotnet/net-framework-4-5-2-4-6-4-6-1-will-reach-end-of-support-on-april-26-2022/) and [this](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-framework)).

There's not really anything in .net 4.7.1 that we need?